### PR TITLE
LitElement conversion part 1: cleanup

### PR DESCRIPTION
**Pull request by web-padawan**
Connected to #128

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,69 +1,15 @@
...
-      magi p3-convert --out . --import-style=name &&
-      yarn install --flat &&
-      if [[ "$TRAVIS_EVENT_TYPE" != "pull_request" && "$TRAVIS_BRANCH" != quick/* ]]; then
-        wct --npm --env saucelabs;
-      else
-        xvfb-run -s '-screen 0 1024x768x24' wct --npm;
-      fi;
-    fi
+
+- npm run check:version
```
Comment (yuriy-fix) [inline .travis.yml]: Seems to be different to what is used in the [`vaadin-tabs`](https://github.com/vaadin/vaadin-tabs/blob/bb1cf73d9cb8a27cb6caea54b238cf860f24fb52/.travis.yml#L12):
```
- if [[ "$TRAVIS_EVENT_TYPE" != "pull_request" ]]; then
    npm run test:sauce && npm run test:visual;
  else
    npm run check:version &&
    npm run lint &&
    npm test;
  fi
```

File: README.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+[![Build Status](https://travis-ci.org/vaadin/vaadin-app-layout.svg?branch=master)](https://travis-ci.org/vaadin/vaadin-app-layout)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/vaadin/web-components?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Published on Vaadin Directory](https://img.shields.io/badge/Vaadin%20Directory-published-00b4f0.svg)](https://vaadin.com/directory/component/vaadinvaadin-app-layout)
+[![Stars on vaadin.com/directory](https://img.shields.io/vaadin-directory/stars/vaadinvaadin-app-layout.svg)](https://vaadin.com/directory/component/vaadinvaadin-app-layout)
+
+> ⚠️ This is a pre-release version built with [`LitElement`](https://github.com/Polymer/lit-element), part of the [next generation of Vaadin web components](https://vaadin.com/blog/next-generation-vaadin-components).
+>
+> Looking for Vaadin 14 compatible version? Please see the following branches:
+> - [1.0 branch](https://github.com/vaadin/vaadin-app-layout/tree/2.0)
+> - [1.1 branch](https://github.com/vaadin/vaadin-app-layout/tree/2.1)
```
Comment (DiegoCardoso) [inline README.md]: Branch names don't match with URLs. There's also `1.0` branch.

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "lib": [ "esnext", "es2017", "dom" ],
```
Comment (yuriy-fix) [inline tsconfig.json]: `es2017` update to align with `"ecmaVersion": 2018` 🤔

File: .travis.yml (hunk 1 above)
Comment (web-padawan) [reply .travis.yml]: Will be re-enabled in the following PRs.

File: tsconfig.json (hunk 3 above)
Comment (web-padawan) [reply tsconfig.json]: Done.

File: README.md (hunk 2 above)
Comment (web-padawan) [reply README.md]: Thanks, done.

IMO it's enough to only mention V14 branches, as 1.0 is actually V10

**Pull request merged**